test/models/transforms/test_input.py

```diff
@@ -259,7 +261,9 @@ def test_normalize(self):
                     [X.min(dim=-2, keepdim=True)[0], X.max(dim=-2, keepdim=True)[0]],
                     dim=-2,
                 )
-                self.assertAllClose(nlz.bounds, expected_bounds)
+                atol = 1e-6 if dtype is torch.float32 else None
```

Comment:
What do you think of dividing the exponents here by two for double precision, rather than using the defaults implicitly through `None`?
```suggestion
                atol = 1e-6 if dtype is torch.float32 else 1e-12
```

Reply:
makes sense -- the defaults are `rtol=1e-5`, `atol=1e-8`, so that change is stricter